cli/main.py

```diff
@@ -130,7 +124,7 @@ def get_settings_files(self):
         settings_files = []
 
         # first take all input files from args
-        for input_file in self.args['input'] or []:
+        for input_file in self.args.get('input', []) or []:
```

Comment:
When 'input' is present in spec file but not provieded via CLI, argparse will assign it to None. 
So self.args['input'] == None.